packages/devtools-source-map/src/source-map.js

```diff
@@ -68,9 +68,8 @@ async function getGeneratedLocation(
 
   return {
     sourceId: generatedSourceId,
-    line: line,
-    // Treat 0 as no column so that line breakpoints work correctly.
-    column: column === 0 ? undefined : column
+    line,
+    column
```

Comment:
IIUC, this change and the corresponding change in `getOriginalLocation` are the critical bits of this patch, and the rest is updating tests, etc.

If that's correct then this patch seems good to me.  It's hard to know offhand if this will have any negative fallout, but I think less weirdness at this layer is a better way generally.

I don't know if it matters here but it's worth noting that the mapping isn't necessarily 1:1.  For example, if a sourcemap generator did inlining, then you could see multiple generated locations with the same source location.  I think sourcemaps can't represent the inverse, but conceptually you could have, say, identical code folding and then a single generated location would have multiple original locations.  So anyway the most that can be said is that when mapping from generated to original to generated, you'll be guaranteed that your initial value will appear in the result set.

Reply:
> then you could see multiple generated locations with the same source location

Yeah, i was thinking that too. I feel like this reduces the number of divergences, but it there could always be an onto relationship where many many G maps to one O location.